src/libs/Notification/PushNotification/index.native.js

```diff
@@ -48,6 +48,30 @@ function pushNotificationEventCallback(eventType, notification) {
     action(payload);
 }
 
+/**
+ * Register push notification callbacks. This is separate from namedUser registration because it needs to be executed
+ * from a headless JS process, outside of any react lifecycle.
+ */
+function init() {
```

Comment:
Does this headless JS process work by running in the background after a user has closed the app?

Reply:
Yep!